src/app/dashboard/directory-dashboard/directory-items/directory-item/directory-item.component.ts

```diff
@@ -39,8 +39,7 @@ export class DirectoryItemComponent implements OnInit {
   ngOnInit() {    
     this.directory.selected = false;
     this.updateDashboardDataSub = this.dashboardService.updateDashboardData.subscribe(val => {
-      this.directory = this.directoryDbService.getById(this.directory.id);
-      this.directory.selected = false;
```

Comment:
we will still want this selected value set to false when this behavior subject is fired

Reply:
in order to have the `this.directory.selected = false;` it needs to be in an If statement checking if that directory exists or else an error gets thrown